### PR TITLE
docs(atoms): FRMW-582-text inputs atom page

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-582

### LGTMs
- [ ] Dev LGTM
- [ ] Dev LGTM
- [ ] Design LGTM
- ~~[ ] QE LGTM~~ (Documentation only)

---
Add text input atom documentation page.

### CODE REVIEW

**Comment by 100stacks** [comment]
@bry4n, so according to @mention-bot you are still on the hook for the `Framework`. :wink:

File: demo/examples/text.input.disabled.plain.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<form rx-form name="demoDisabledTextboxForm" style="max-width: 300px">
+  <rx-form-section>
+    <rx-field>
+      <rx-field-name>IP Address:</rx-field-name>
+      <rx-field-content>
+        <rx-input>
+          <input name="ipaddress"
+            type="text"
+            disabled
+            placeholder="127.0.0.1" />
```
Comment (bry4n) [inline demo/examples/text.input.disabled.plain.html]: indent `type`, `disabled`, and `placeholder` to match the `name` position
```
<input name="ipaddress"
       type="text"
       placeholder="127.0.0.1"
       disabled />
````

File: demo/examples/text.input.disabled.plain.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<form rx-form name="demoDisabledTextboxForm" style="max-width: 300px">
```
Comment (bry4n) [inline demo/examples/text.input.disabled.plain.html]: are we allowed to put `style` attribute in demo?

Comment (glynnis) [inline demo/examples/text.input.disabled.plain.html]: Eww, yeah, why is this here?

File: demo/examples/text.input.plain.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<form rx-form name="demoPlainTextboxForm" style="max-width: 300px">
```
Comment (glynnis) [inline demo/examples/text.input.plain.html]: Do we really need `style` here, or can it be removed (or is there some CSS that we should change up elsewhere)? Ditto https://github.com/rackerlabs/encore-ui/pull/1593/files#r55743148.

File: demo/examples/text.input.scroll.plain.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<form rx-form name="demoPlainScrollTextboxForm" style="max-width: 300px">
```
Comment (glynnis) [inline demo/examples/text.input.scroll.plain.html]: Do we really need `style` here, or can it be removed (or is there some CSS that we should change up elsewhere)? Ditto https://github.com/rackerlabs/encore-ui/pull/1593/files#r55743148.

File: demo/examples/text.input.validation.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
+<form rx-form name="demoValidationTextboxForm" style="max-width: 700px">
```
Comment (glynnis) [inline demo/examples/text.input.validation.html]: Do we really need `style` here, or can it be removed (or is there some CSS that we should change up elsewhere)? Ditto https://github.com/rackerlabs/encore-ui/pull/1593/files#r55743148.

File: demo/examples/text.input.disabled.plain.html (hunk 1 above)
Comment (CITguy) [inline demo/examples/text.input.disabled.plain.html]: I'm for keeping attributes indented at the same level, but I'd put `name="ipaddress"` on a new line below the `<input` so we can apply our 2 space HTML indentation standard.

```
<input
  name="ipaddress"
  type="text"
  disabled
  placeholder="127.0.0.1" />
```

File: demo/examples/text.input.scroll.plain.html (hunk 4 above)
Comment (CITguy) [inline demo/examples/text.input.scroll.plain.html]: I'd apply a class to be styled via `demo/examples/text.input.scroll.plain.less`

File: demo/examples/text.input.validation.html (hunk 5 above)
Comment (CITguy) [inline demo/examples/text.input.validation.html]: I'd apply a class to be styled via `demo/examples/text.input.validation.less`

File: demo/examples/text.input.disabled.plain.html (hunk 1 above)
Comment (bry4n) [inline demo/examples/text.input.disabled.plain.html]: :+1:

File: demo/examples/text.input.disabled.plain.html (hunk 2 above)
Comment (CITguy) [inline demo/examples/text.input.disabled.plain.html]: I'd apply a class to be styled via `demo/examples/text.input.disabled.plain.less`

**Comment by 100stacks** [comment]
@CITguy @glynnis, holding off on any updates until we hear back from the meeting next week...unless you think it's still a valid PR/story. :100:

**Comment by CITguy** [comment]
If we decide to change nomenclature, this would make more sense as examples within `rxForm` rather than `demo/`. Marking DO NOT MERGE.

**Comment by 100stacks** [comment]
Based on recent discussions, I'm closing this PR since we're shifting this documentation location.
